File: code-coaches/easy/multiples/Multiples.cs
using System;

class Multiples
{
    static int SumMultiples(int num, params int[] multiples)
    {
        int sumOfMultiples = 0;
        for (int i = 0; i < num; i++)
        {
            foreach (int multiple in multiples)
            {
                if (i % multiple == 0)
                {
                    sumOfMultiples += i;
                    break;
                }
            }
        }
        return sumOfMultiples;
    }

    static void Main(string[] args)
    {
        int num = int.Parse(Console.ReadLine());
        Console.WriteLine(SumMultiples(num, 3, 5));
    }
}

File: code-coaches/easy/multiples/Multiples.java
import java.util.Scanner;

public class Multiples {
    public static int sumMultiples(int num, int... multiples) {
        int sumOfMultiples = 0;
        for (int i = 0; i < num; i++) {
            for (int multiple : multiples) {
                if (i % multiple == 0) {
                    sumOfMultiples += i;
                    break;
                }
            }
        }
        return sumOfMultiples;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();

        System.out.println(sumMultiples(num, 3, 5));

        sc.close();
    }
}
